src/alcotest.ml

```diff
@@ -383,7 +390,9 @@ let show_result t result =
     in
     let full_logs ppf =
       if t.verbose then Fmt.string ppf ""
-      else Fmt.pf ppf "The full test results are available in `%s`.\n" t.test_dir
+      else
+        Fmt.pf ppf "The full test results are available in `%s`.\n"
+          (output_dir t)
```

Comment:
Not directly related to that PR but #135 points that this location is in general wrong. Might be a good occasion to fix that too, but I am fine if this is done in a different PR too!

Reply:
Good call out, see the comment I left on #135 for my analysis of the issue. The summary is that I believe `dune` alters the current working directory and as the default `test_dir` is a relative rather than an absolute path it appears to be wrong (the problem being that in the context of the test runner it is correct).

My fix (pushed in `fbfb395`) is to set this default `test_dir` to an absolute path whose prefix is the current working directory. This means semantically there is no change but when `t.test_dir` is printed to the user it is explicit about the current working directory (and just the entire location). It also means the current working directory is taken once at the start of the run rather than implicitly picking it up every time that `t.test_dir` was referenced in some I/O.